README.rst

```diff
@@ -216,3 +217,14 @@ Documentation for other versions
 * `v0.1.1 <https://www.pygmt.org/v0.1.1>`__
 * `v0.1.0 <https://www.pygmt.org/v0.1.0>`__
 * `v0.0.1a0 <https://www.pygmt.org/0.0.1a0>`__
+
+Compatibility with Python and GMT versions
+------------------------------------------
+
+======= ========== =========
+PyGMT   GMT        Python
+======= ========== =========
+0.2.1   >=6.1.1    3.6 - 3.9
```

Comment:
Since we went no-arch in v0.2.1 (see #690), should the Python compatibility be `>=3.6`? Not sure how to test on Python 3.10 but it should technically work.

Reply:
Do Python 3.6 codes always work with future Python versions?

Reply:
It should? I think async/await became reserved names in 3.7 but PyGMT doesn't use those, or any advanced stuff really. Things should be safe until Python 4 at least.

Reply:
OK, changed to ">=3.6".